src/bp-templates/bp-nouveau/includes/xprofile/functions.php

```diff
@@ -43,3 +43,55 @@ function bp_nouveau_xprofile_enqueue_scripts() {
 	}
 	wp_enqueue_script( 'bp-nouveau-xprofile' );
 }
+
+
+/**
+ * Return profile header buttons
+ *
+ * @since BuddyBoss 1.5.1
+ *
+ * @return mixed|void
+ */
+
+function bb_profile_header_buttons() {
```

Comment:
@shohelbuddyboss this functions should be in `bp-templates/bp-nouveau/includes/customizer.php`

Reply:
moved and renamed, new function name `bp_get_profile_header_buttons();`